source/Personality.cpp

```diff
@@ -49,7 +49,11 @@ namespace {
 	const int TARGET = (1 << 25);
 	const int MARKED = (1 << 26);
 	const int LAUNCHING = (1 << 27);
-	
+	const int ROVING = (1 << 28);
+	const int CLOAKING = (1 << 29);
+	const int EVASIVE = (1 << 30);
+	const int BORED = (1 << 31);
```

Comment:
Seems like we're hitting the limit of personalities. interesting

Reply:
Yeah, Derpy mentioned that in one of his streams. Might wanna change that to a `long` instead of just an `int`?